test/hotspot/jtreg/compiler/codegen/TestGCMLoadPlacement.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Since you opened the PR in the old year, you can probably just append 2025 here as well instead of replacing it.

```suggestion
 * Copyright (c) 2024, 2025, Oracle and/or its affiliates. All rights reserved.
```